src/php/README.md

```diff
@@ -28,10 +28,39 @@ $ curl -O http://pear.php.net/go-pear.phar
 $ sudo php -d detect_unicode=0 go-pear.phar
 ```
 
-## Build from Homebrew
+## Quick Install
 
-On Mac OS X, install [homebrew][]. On Linux, install [linuxbrew][]. Run the following command to
-install gRPC.
+**Linux (Debian):**
+
+Add [debian unstable][] (sid) to your `sources.list` file. Example:
```

Comment:
You could put it in https://github.com/grpc/grpc/blob/master/README.md or put it in `INSTALLATION` and link to a section (via an anchor, e.g. `https://github.com/grpc/grpc/blob/master/INSTALL#foo`).

This may rely on converting INSTALL to valid markdown